enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -6,6 +6,57 @@ use sgx_types::{ssa::StateSaveArea, tcs::Tcs};
 
 use core::ops::Range;
 
+// ===== Temporary nolibc functions ================
+
+enumerate::enumerate! {
+    #[derive(Copy, Clone)]
+    pub enum ArchPrctlTask: u64 {
+        ArchSetGs = 0x1001,
+        ArchSetFs = 0x1002,
+        ArchGetFs = 0x1003,
+        ArchGetGs = 0x1004,
+    }
+}
```

Comment:
This doesn't belong here.

Reply:
I just tried to put it back in `nolibc` and it resulted in the same linking error. I don't think relying on `nolibc` is a good idea at the moment.